contrib/curl/premake5.lua

```diff
@@ -45,6 +45,7 @@ project "curl-lib"
 			"/usr/local/share/certs/ca-root-nss.crt",
 			"/etc/certs/ca-certificates.crt",
 			"/etc/ssl/cert.pem",
+			"/etc/ssl/cacert.pem",
```

Comment:
Is it part of this PR?

Reply:
Yes, that's where the `web/ca-bundle` package installed the CA certs.